packages/lambda-server/netlify.toml

```diff
@@ -0,0 +1,4 @@
+[build]
+  command = "npm run build && npm run build" # the command you run to build this file
+  functions = "built-lambda-server" # netlify-lambda builds to this folder AND Netlify reads functions from here
+  
```

Comment:
let's have also all necessary config params tu deploy and run s3-graphql demo

[[plugins]]


[build.environment]
  NODE_VERSION = "14"
  NPM_FLAGS = "--prefix=/dev/null"
  NODE_OPTIONS = "--max_old_space_size=4096"

[[redirects]]
  from = "/*"
  to = "/index.html"
  status = 200

[[headers]]
  # Define which paths this specific [[headers]] block will cover.
  for = "/*"
    [headers.values]
    Access-Control-Allow-Origin = "*"

[build]
  command = "npx pnpm i --store=node_modules/.pnpm-store && npx pnpm run build"
  publish = "dist"

Reply:
https://github.com/iteria-app/lowcode-monorepo/issues/547